solutions/CF Good Subarrays.mdx

```diff
@@ -0,0 +1,54 @@
+---
+id: cf-1398C
+source: CF
+title: Good Subarrays
+author: Jesse Choe
+---
+[Official Editorial](https://codeforces.com/blog/entry/81506)
+
+<LanguageSection>
+
+<CPPSection>
+
+## Solution - Prefix Sums + Math
+
+```cpp
+#include <bits/stdc++.h>
+
+using namespace std;
+using ll = long long;
+using pi = pair<ll, ll>;
+
+void solve(){
```

Comment:
spacing is inconsistent; either space all operators on both sides or don't space any

Reply:
like you have `sumDist[dp[i] - i]++;` and also `dp[i-1]` so just make the spacing the same (i didn't mean the newlines)